File: Daily Coding/python/no_of_digits_sum.py
#Get two numbers as input. Find the sum of the numbers.
#If the number of digits in the sum is greater than the number of digits in the first number then print only the first number, else print the sum
def find_digits(num):
    digits = 0
    while num > 0:
        num //= 10
        digits += 1
    return digits

print("Enter the numbers: ")
num1 = int(input())
num2 = int(input())
num_digits = find_digits(num1)
total = num1 + num2
sum_digits = find_digits(total)
if sum_digits == num_digits:
    print(total)
else:
    print(num1)

File: Daily Coding/c/no_of_digits_sum.c
/*
Get two numbers as input. Find the sum of the numbers.
If the number of digits in the sum is greater than the number of digits in the first number then print only the first number, else print the sum
*/
#include <stdio.h>
int is_prime(int num)
{
    int i;
    for(i=3; i<=(num/2); i++)
    {
        if(num%i == 0)
            return 0;
    }
    return 1;
}
int main()
{
    int num, i;
    printf("Enter the number: ");
    scanf("%d",&num);
    for(i=3; i<=num; i++)
    {
        if(is_prime(i) && is_prime(num-i))
        {
            printf("%d %d",i,num-i);
            break;
        }
    }
    return 0;
}
